packages/accounts/src/kernel-zerodev/provider.ts

```diff
@@ -0,0 +1,46 @@
+import type { HttpTransport } from "viem";
+import {
+  type AccountMiddlewareFn,
+  deepHexlify,
+  resolveProperties,
+  SmartAccountProvider,
+} from "@alchemy/aa-core";
+
+export class KernelAccountProvider extends SmartAccountProvider<HttpTransport> {
+  gasEstimator: AccountMiddlewareFn = async (struct) => {
+    const request = deepHexlify(await resolveProperties(struct));
+    const estimates = await this.rpcClient.estimateUserOperationGas(
+      request,
+      this.entryPointAddress
+    );
+
+    estimates.verificationGasLimit =
+      (BigInt(estimates.verificationGasLimit) * 130n) / 100n;
```

Comment:
oh interesting, where's the extra 30% overhead come from?

Reply:
curious to learn about this 30% overhead too